rbm_train.py

```diff
@@ -157,14 +160,22 @@ def imgshow(file_name, img):
     re_mean = np.mean(full_reconstruction_error)
     loss_mean = np.mean(loss_)
     free_energy_mean = np.mean(free_energy_)
-    pbar.set_description("Epoch %3d - Loss %8.5f - RE %5.3g " % (epoch, loss_mean, re_mean))
 
-    loss_file.write(str(epoch) + "\t" + str(loss_mean) + "\t" +  str(free_energy_mean) + "\t" + str(re_mean) + "\n")
+    log_likelyhood_mean = free_energy_mean - logz
+
```

Comment:
we could then just use above function here

Reply:
also for the sake of consistency, might be worth always spelling likelihood with an i